src/hooks/useLongPress.ts

```diff
@@ -41,6 +42,7 @@ const useLongPress = (
         clientCoords.current = { x: e.touches?.[0]?.clientX, y: e.touches?.[0]?.clientY }
       }
       onTouchStart?.()
+      Haptics.selectionStart()
```

Comment:
This should go next to `onLongPressStart`, where the long press actually starts.

`start` only starts the timer.